htdocs/compta/facture/class/facture.class.php

```diff
@@ -514,6 +514,7 @@ public function create(User $user, $notrigger = 0, $forceduedate = 0)
 			$this->id = $this->db->last_insert_id(MAIN_DB_PREFIX.'facture');
 
 			// Update ref with new one
+			//if (empty($this->ref))	$this->ref='(PROV'.$this->id.')';
```

Comment:
With current version of Dolibarr, using a numbering module for invoice ref is mandatory.
Why do you need this ?

Reply:
I saw the code didn't conform with description (It said: _this->ref can be set or empty. If empty, we will use "(PROV999)"_). And my module change the **ref** field, this fix isn't essential, but it would be useful.

What about the fix in card.php?, Is it right?